core/test/test_server.hh

```diff
@@ -42,9 +42,9 @@ class test_server {
 
   void init();
   void run();
-  void stop() { _ctx->stop(); };
+  void stop() { std::unique_lock<std::mutex> lock(_m); if (_initialised) _ctx->stop(); };
```

Comment:
Is it possible to have a mutex name saying what it locks?
Usually, if I want to access to a variable toto, I create a mutex toto_m.